src/components/Background.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import '../index.css';
```

Comment:
Change to absolute path

Reply:
He said he's having trouble with this... please help him out AND apply " rule here if we are applying in backend... or we should use '... since most files/demos i see use '...

Reply:
Or maybe we can use ' for imports?